src/oss/confluent.sh

```diff
@@ -542,6 +544,46 @@ wait_connect() {
     wait_process_up "${pid}" 4000 || echo "Kafka Connect failed to start"
 }
 
+start_ksql() {
+    local service="ksql"
+    is_running "kafka" "false" \
+        || die "Cannot start ksql, Kafka Server is not running. Check your deployment"
+    export_service_env "KSQL_"
```

Comment:
This actually should not be called. This is only for services that use `KAFKA_` prefix for some of their environment variables. Since ksql does always use `KSQL_` we shouldn't call this function here.